package.json

```diff
@@ -23,12 +23,12 @@
     "node": ">=20.x"
   },
   "dependencies": {
-    "class-variance-authority": "^0.7.0",
+    "class-variance-authority": "^0.7.1",
     "classnames": "^2.5.1",
     "dayjs": "^1.11.13",
-    "next": "15.0.2",
-    "react": "19.0.0-rc-02c0e824-20241028",
-    "react-dom": "19.0.0-rc-02c0e824-20241028",
+    "next": "15.0.3",
```

Comment:
We should probably discuss whether we really want to pin the version

Reply:
why not?